jobs/build/promote-assembly/Jenkinsfile

```diff
@@ -184,34 +191,6 @@ node {
     buildlib.registry_quay_dev_login()  // chances are, earlier auth has expired
 
     def justifications =release_info.justifications ?: []
-    def quay_url = "quay.io/openshift-release-dev/ocp-release"
-    def client_type = 'ocp'
-    if ((release_info.type == "candidate" && !release_info.assembly.startsWith('rc.')) || release_info.type == "custom" || release_info.type == "preview") {
-        // Feature candidates and custom releases and .next "preview" releases use client_type ocp-dev-preview and beta2 signing key.
-        client_type = 'ocp-dev-preview'
-    }
-    stage("mirror binaries") {
-        if (params.SKIP_MIRROR_BINARIES) {
-            echo "Skip mirroring binaries."
-            return
-        }
-        release_info.content.each { arch, info ->
-            echo "Mirroring client binaries for $arch"
-            def dest_release_tag = release.destReleaseTag(release_info.name, arch)
-            // Currently a multi/heterogeneous release payload has a modified release name to workaround a Cincinnati issue.
-            // Using the real per-arch release name in $info instead of the one defined by release artists.
-            def release_name = info.metadata.version
-            if (!params.DRY_RUN) {
-                if (arch != "multi") {
-                    release.stagePublishClient(quay_url, dest_release_tag, release_name, arch, client_type)
```

Comment:
I don't see this func getting removed.. is anything else using it?
It would be helpful to compare them side by side

Reply:
I saw it in pre-release job and release job, but seems theses jobs are not in use